### DISCUSSION TITLE
I need to learn the basics of github

### DISCUSSION THREAD

**Discussion by tiijaay1** [community/community#83534] [New to GitHub] [answered]
Obviously i am a newbie in this field , and for me to come late is better than never

**Comment by codesmith-emmy** [comment, marked as answer]
Of course, better late than never. And I assure you your zeal to learn will pay off handsomely.

You can start your GitHub journey with these free resources:

1. [Getting Started with Git and GitHub on Coursera](https://www.coursera.org/learn/getting-started-with-git-and-github?utm_medium=sem&utm_source=gg&utm_campaign=B2C_EMEA_ibm-backend-development_ibm_FTCOF_professional-certificates_country-UK-country-GB&campaignid=20858197900&adgroupid=156245797669&device=m&keyword=&matchtype=&network=g&devicemodel=&adposition=&creativeid=684297720233&hide_mobile_promo)
2. [Git and GitHub for Beginners - Crash Course on YouTube](https://m.youtube.com/watch?v=RGOj5yH7evk&pp=ygUPR2l0aHViIHR1dG9yaWFs) 

Good luck! 👍🏽

**Reply by tiijaay1** [reply]
Thanx so much for you kind reply i will check those lessons
